addons/sms/tools/sms_api.py

```diff
@@ -91,7 +91,7 @@ def _get_sms_api_error_messages(self):
             _('Register now.')
         )
 
-        return {
+        return super()._get_sms_api_error_messages() & {
```

Comment:
If you want to do that it's `|` :disguised_face:  (but I see you converted to update anyway)